doc/nrf/ug_tfm.rst

```diff
@@ -102,26 +107,27 @@ The logs arrive on different COM ports on the host PC.
    * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
      Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
 
-, at this time, not all SPM functions are available in TF-M.
+Migrating from Secure Partition Manager to Trusted Firmware-M
+*************************************************************
+
+The interface to TF-M is different from the interface to SPM.
+Due to that, the application code that uses the SPM :ref:`lib_secure_services` needs to be ported to use TF-M instead.
 
-Migrating from SPM:
-*******************
+The following SPM services can be replaced by TF-M:
 
-Application code that uses SPM :ref:`lib_secure_services` needs to be ported to use TF-M instead because the interface to TF-M is different.
+* ``spm_request_system_reboot`` uses ``tfm_platform_system_reset``.
```

Comment:
The meaning of this is wrong now.

Maybe:
``spm_request_random_number`` use ``psa_generate_random`` or ``entropy_get_entropy`` instead?
or:
``spm_request_random_number`` replaced by ``psa_generate_random`` or ``entropy_get_entropy``.

Reply:
Ah, I see! :)
How about:
* Replace ``spm_request_random_number`` with ``psa_generate_random`` or ``entropy_get_entropy``.